modules/templates/helper.go

```diff
@@ -238,6 +239,15 @@ func DotEscape(raw string) string {
 	return strings.ReplaceAll(raw, ".", "\u200d.\u200d")
 }
 
+func Iif(condition bool, vals ...any) any {
```

Comment:
This has been implemented in `util/util.go:217`

Reply:
No, they are different, the `util` one can't be used in templates.